src/uv-common.c

```diff
@@ -225,6 +225,30 @@ int uv_ip6_name(const struct sockaddr_in6* src, char* dst, size_t size) {
   return uv_inet_ntop(AF_INET6, &src->sin6_addr, dst, size);
 }
 
+int uv_if_indextoname(unsigned int ifindex, char* ifname) {
+#ifdef _WIN32
+  NET_LUID InterfaceLuid;
+  char InterfaceName[NDIS_IF_MAX_STRING_SIZE + 1];
+  if (!ConvertInterfaceIndexToLuid(ifindex, &InterfaceLuid)) {
```

Comment:
I don't like having Windows APIs in uv-common, let's split this and have it in src/unix/core.c and src/win/util.c please.